packages/data-service/src/instance-detail-helper.ts

```diff
@@ -108,12 +107,12 @@ export type InstanceDetails = {
 };
 
 export async function getInstance(
-  client: MongoClient
+  client: MongoClient,
+  uri: string
```

Comment:
Is there a better way for me to get the connection uri here? MongoClient is in scope but does not provide an accessor to fetch the connection string.

Reply:
I think `MongoClient`s only store the parsed connection string options, not the original string, unfortunately 😐